wpinet/src/main/native/include/wpinet/UDPClient.h

```diff
@@ -8,10 +8,18 @@
 #include <span>
 #include <string>
 #include <string_view>
+#include <span>
 
 #include <wpi/SmallVector.h>
 #include <wpi/mutex.h>
 
+#ifdef _WIN32
+#include <WinSock2.h>
+#pragma comment(lib, "Ws2_32.lib")
+#else
+#include <netinet/in.h>
+#endif
```

Comment:
I really don't love pulling these in so I can get sockaddr_in in the api. I think this can probably become a forward declaration, or can be removed entirely. Or maybe wanting to pass a sockaddr_in directly to send/get a sockaddr_in directly from recieve is premature optimization?

Reply:
The pragma lib shouldn’t be in the header. It should just be in the cpp file.

Reply:
As for stockaddr in, I’d very much prefer to make our own abstraction with a few forward declarations to allow building a conversion function.

Reply:
I would prefer we use libuv for this.  We can set up a separate loop, and it has all the right wrappers for Windows and Linux already.

Reply:
Libuv-ified!